packages/pds/src/auth-verifier.ts

```diff
@@ -522,7 +539,7 @@ export class AuthVerifier {
     const { did, scope, token, audience } = await this.validateBearerToken(
       ctx,
       scopes,
-      { audience: this.dids.pds },
+      { audience: this.dids.pds, typ: 'at+jwt' },
```

Comment:
Will these new `typ` checks break existing sessions?

Reply:
Ah, I see you handle this case 👍

Reply:
Yep, missing `typ` are allowed (for now)